src/internal/bind-to-namespace.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Bind all methods of a class to the class instance.
+ *
+ * @param moduleClass The class to bind the methods of.
+ * @param that The class instance to bind the methods to.
+ */
+export function bindToNamespace<TClass extends { new (...args: any[]): any }>(
```

Comment:
We could reduce the number of `any`s here:

```suggestion
export function bindToNamespace<TClass extends { new (faker: Faker): any }>(
```

Reply:
Or even just:

```suggestion
export function bindToNamespace<TClass extends { new (): any }>(
```